scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -408,7 +408,7 @@ private static void CleanOutputDir(string directory)
 
         private static void RemovePdbsFromDir(string directory)
         {
-            FS.RmFilesInDirRecursive(directory, "*.pdb");
+            //FS.RmFilesInDirRecursive(directory, "*.pdb");
```

Comment:
don't leave comments. either delete the file or not. if this is temporarily, remove to removed it.